Safeguard.SecureStore/SafeguardSecureStore/SafeguardContextBuilder.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Newtonsoft.Json;
+using System;
+using UiPath.Orchestrator.Extensibility.SecureStores;
+using UiPath.Orchestrator.SafeguardSecureStore;
+
+namespace UiPath.Orchestrator.Extensions.SecureStores.Safeguard
+{
+    public class SafeguardContextBuilder
+    {
+        private SafeguardContext _context;
+                
+        public SafeguardContextBuilder FromJson(string json)
+        {
+            json = json ?? throw new SecureStoreException(
+                SecureStoreException.Type.InvalidConfiguration,
+                SafeguardUtils.GetLocalizedResource(nameof(Resource.SafeguardJsonInvalidOrMissing), json));
+
+            try
+            {
+                _context = JsonConvert.DeserializeObject<SafeguardContext>(json);
+            }
+            catch (Exception)
+            {
+                throw new Exception(
+                        SafeguardUtils.GetLocalizedResource(nameof(Resource.JsonDeserializationIssue), json, _context.SafeguardAppliance, _context.SafeguardCertThumbprint));
+                    
+            }
+
+            _context = _context ?? throw new SecureStoreException(
```

Comment:
@VVargaOI what does this line do? Is there ever a situation where this would throw the exception, given that if the initialisation on line 20 ever fails, you get an exception from line 24?